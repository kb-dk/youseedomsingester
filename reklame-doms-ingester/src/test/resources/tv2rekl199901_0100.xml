<?xml version="1.0" encoding="UTF-8" standalone="no"?><PBCoreDescriptionDocument xmlns="http://www.pbcore.org/PBCore/PBCoreNamespace.html">
    <pbcoreAssetType>Tv2reklamefilm</pbcoreAssetType>
    <pbcoreIdentifier>
        <identifier>tv2rekl199901_0100</identifier>
        <identifierSource>Statens Mediesamling TV2 reklamer 1988-2005</identifierSource>
    </pbcoreIdentifier>
    <pbcoreTitle>
        <title>Fanta</title>
    </pbcoreTitle>
    <pbcoreTitle>
        <title>Fanta Drop - Fanta Paris/Share the Fun. Wherever</title>
        <titleType>alternative</titleType>
    </pbcoreTitle>
    <pbcoreSubject>
        <subject>Drikkevarer</subject>
        <subjectAuthorityUsed>Statens Mediesamling TV2 reklamer 1988-2005 vocabulary top-level</subjectAuthorityUsed>
    </pbcoreSubject>
    <pbcoreSubject>
        <subject>LÃ¦skedrikke</subject>
        <subjectAuthorityUsed>Statens Mediesamling TV2 reklamer 1988-2005 vocabulary bottom-level</subjectAuthorityUsed>
    </pbcoreSubject>
    <pbcoreDescription>
        <description/>
    </pbcoreDescription>
    <pbcoreGenre>
        <genre>Advertising</genre>
        <genreAuthorityUsed>EBUContentGenre</genreAuthorityUsed>
    </pbcoreGenre>
    <pbcoreCreator>
        <creator>Coca Cola Danmark A/S</creator>
        <creatorRole>Creator</creatorRole>
    </pbcoreCreator>
    <pbcoreCreator>
        <creator>Para Film</creator>
        <creatorRole>Producer</creatorRole>
    </pbcoreCreator>
    <pbcoreCreator>
        <creator>Tarsan Film</creator>
        <creatorRole>Production Unit</creatorRole>
    </pbcoreCreator>
    <pbcoreInstantiation>
        <pbcoreFormatID>
            <formatIdentifier>tv2rekl199901_0100</formatIdentifier>
            <formatIdentifierSource>Statens Mediesamling TV2 reklamer 1988-2005</formatIdentifierSource>
        </pbcoreFormatID>
        <dateIssued>1999-01-05+0100</dateIssued>
        <formatLocation>tv2rekl199901_0100.mpg</formatLocation>
        <formatDuration>00:00:30</formatDuration>
        <language>dan</language>
    </pbcoreInstantiation>
    <pbcoreExtension>
       <extension>locationoforiginalproduction: Dansk</extension>
       <extensionAuthorityUsed>Statens Mediesamling TV2 reklamer 1988-2005</extensionAuthorityUsed>
   </pbcoreExtension>
</PBCoreDescriptionDocument>