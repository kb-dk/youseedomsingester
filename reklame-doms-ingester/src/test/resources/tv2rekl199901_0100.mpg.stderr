<ffprobe:ffprobeStdErrorOutput xmlns:ffprobe='http://www.ffmpeg.org/schema/ffprobe/stderr'>
ffprobe version 0.10.5 Copyright (c) 2007-2012 the FFmpeg developers
  built on Oct  4 2012 19:17:43 with gcc 4.7.2 20120921 (Red Hat 4.7.2-2)
  configuration: --prefix=/usr --bindir=/usr/bin --datadir=/usr/share/ffmpeg --incdir=/usr/include/ffmpeg --libdir=/usr/lib64 --mandir=/usr/share/man --arch=x86_64 --extra-cflags='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic' --enable-bzlib --disable-crystalhd --enable-gnutls --enable-libass --enable-libcdio --enable-libcelt --enable-libdc1394 --disable-indev=jack --enable-libfreetype --enable-libgsm --enable-libmp3lame --enable-openal --enable-libopenjpeg --enable-libpulse --enable-librtmp --enable-libschroedinger --enable-libspeex --enable-libtheora --enable-libvorbis --enable-libv4l2 --enable-libvpx --enable-libx264 --enable-libxvid --enable-x11grab --enable-avfilter --enable-postproc --enable-pthreads --disable-static --enable-shared --enable-gpl --disable-debug --disable-stripping --shlibdir=/usr/lib64 --enable-runtime-cpudetect
  libavutil      51. 35.100 / 51. 35.100
  libavcodec     53. 61.100 / 53. 61.100
  libavformat    53. 32.100 / 53. 32.100
  libavdevice    53.  4.100 / 53.  4.100
  libavfilter     2. 61.100 /  2. 61.100
  libswscale      2.  1.100 /  2.  1.100
  libswresample   0.  6.100 /  0.  6.100
  libpostproc    52.  0.100 / 52.  0.100
Input #0, mpeg, from 'tv2rekl199901_0100.mpg':
  Duration: 00:00:30.24, start: 0.184656, bitrate: 8534 kb/s
    Stream #0:0[0x1e0]: Video: mpeg2video (Main), yuv420p, 720x576 [SAR 16:15 DAR 4:3], 8027 kb/s, 25 fps, 25 tbr, 90k tbn, 50 tbc
    Stream #0:1[0x1c0]: Audio: mp2, 48000 Hz, stereo, s16, 192 kb/s
</ffprobe:ffprobeStdErrorOutput>